Neos.Neos/Classes/Service/HtmlAugmenter.php

```diff
@@ -79,7 +79,7 @@ protected function getHtmlRootElement($html)
             return null;
         }
         // detect whether loadHTML has wrapped plaintext in a p-tag without asking for permission
-        if ($rootElement instanceof \DOMNodeList && $rootElement->item(0)->tagName === 'p' && substr($html, 0, 2) !== '<p') {
+        if ($rootElement instanceof \DOMNodeList && $rootElement->item(0)->tagName === 'p' && !preg_match('/^<p/ui', $html)) {
```

Comment:
Good catch! Also, while you're at it `/^<p\s/i` wouldn't match the (unlikely) case mentioned above